gammapy/modeling/models/spatial.py

```diff
@@ -533,6 +548,12 @@ def evaluate(lon, lat, lon_0, lat_0, r_0, eta, e, phi):
         norm = 1 / (2 * np.pi * minor_axis * r_0 * eta * scipy.special.gamma(2 * eta))
         return (norm * np.exp(-(z ** (1 / eta)))).to("sr-1")
 
+    @property
+    def evaluation_bin_size_min(self):
+        """ Minimal evaluation bin size (`~astropy.coordinates.Angle`)."""
+        return self.r_0.quantity / (1 + 8 * self.eta.value)
```

Comment:
I used a condition similar to the evaluation radius but with a division instead of multiplication. For eta -> 0 it tends to r_0 as for the disk. For eta=0.5 the factor is r_0/4 = sqrt(2) sigma / 4 ~ sigma /3 as for the gaussian.